src/resources/boleto/schema.js

```diff
@@ -11,7 +11,8 @@ const createSchema = {
 
   expiration_date: Joi
     .date()
-    .max(maximumDateForBarcodeCalculation)
+    .when('issuer', { is: 'boleto-api-bradesco-shopfacil', then: Joi.date().max(maximumDateForBarcodeCalculation) })
```

Comment:
Como a superbowleto só emite caixa e bradesco, o que acha de inverter essa lógica? Só economizaria uma linha kkkkkk
```suggestion
    .when('issuer', { is: 'boleto-api-caixa', then: Joi.date() , otherwise: Joi.date().max(maximumDateForBarcodeCalculation) })
```

Reply:
Ótimo ponto, Jean! Vou subir a mudança já já